apps/main/src/loan/lib/apiCrvusd.ts

```diff
@@ -47,6 +43,74 @@ const DEFAULT_PARAMETERS = {
   loan_discount: '',
 }
 
+function parseUserLoss(userLoss: UserLoanDetails['userLoss']) {
+  const smallAmount = 0.00000001
+  const resp = cloneDeep(userLoss)
+  resp.loss = resp.loss && BN(resp.loss).isLessThan(smallAmount) ? '0' : userLoss.loss
+  resp.loss_pct = resp.loss_pct && BN(resp.loss_pct).isLessThan(smallAmount) ? '0' : userLoss.loss_pct
+
+  return resp
+}
+
+async function fetchChartBandBalancesData(
```

Comment:
Sadly couldn't easily dedupe these for lend and loan. But they might get deleted with JJ's new bands chart.